spec/controllers/snippets/snippets_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+RSpec.describe Snippets::SnippetsController, type: :request do
+  before(:each) do
+    DatabaseCleaner.clean
+  end
+
+  describe 'GET /snippets' do
+    context 'when displaying the user\'s snippets (sorting/filtering disabled)' do
+      let(:user) { create(:user) }
+
+      before(:each) do
+        @first_snippet = create(:first_snippet, user: user)
+        @second_snippet = create(:second_snippet, user: user)
+        @third_snippet = create(:third_snippet, user: user)
+
+        sign_in user
+
+        get '/snippets'
+      end
+
+      it 'returns a successful response' do
+        expect(response).to have_http_status(200)
+      end
+
+      it 'returns the correct number of snippets' do
+        parsed_response = JSON.parse(response.body)
```

Comment:
You can move this line to a global `let` at a higher level, then you don't have to repeat in every test.